lib/netext/httpext/transport.go

```diff
@@ -130,9 +130,6 @@ func (t *transport) measureAndEmitMetrics(unfReq *unfinishedRequest) *finishedRe
 			}
 		}
 
-		if enabledTags.Has(stats.TagMethod) {
-			tags["method"] = unfReq.request.Method
-		}
```

Comment:
I think this needs to stay ... as I am pretty sure if you have `POST -- 301 redirect --> GET` the second request will still be tagged as  `POST` ... I think

Reply:
OK, reverted. We should have a test for this, but I don't want to delay the canary further.

Reply:
Can you start on the test at least, we are not in that much hurry ;)

Reply:
I don't think it's worth delaying the release for this one specific edge case test, but sure, I'll add it.

Reply:
Resolved by af0f5f0c.

You were right, removing this would've broken the second request's `method` tag. 👍